conformant = "Pass"
output = """
File "tuples_type_form.py", line 12, in <module>: Type annotation for t1 does not match type of assignment [annotation-type-mismatch]
File "tuples_type_form.py", line 14, in <module>: Type annotation for t2 does not match type of assignment [annotation-type-mismatch]
File "tuples_type_form.py", line 15, in <module>: Type annotation for t2 does not match type of assignment [annotation-type-mismatch]
File "tuples_type_form.py", line 25, in <module>: Type annotation for t10 does not match type of assignment [annotation-type-mismatch]
File "tuples_type_form.py", line 36, in <module>: Type annotation for t20 does not match type of assignment [annotation-type-mismatch]
File "tuples_type_form.py", line 40, in <module>: Invalid type annotation 'tuple[int, int, Any]'  [invalid-annotation]
File "tuples_type_form.py", line 41, in <module>: Invalid type annotation 'Ellipsis'  [invalid-annotation]
File "tuples_type_form.py", line 42, in <module>: Invalid type annotation 'tuple[Any, int]'  [invalid-annotation]
File "tuples_type_form.py", line 42, in <module>: Invalid type annotation 'Ellipsis'  [invalid-annotation]
File "tuples_type_form.py", line 43, in <module>: Invalid type annotation 'tuple[int, Any, int]'  [invalid-annotation]
File "tuples_type_form.py", line 43, in <module>: Invalid type annotation 'Ellipsis'  [invalid-annotation]
File "tuples_type_form.py", line 44, in <module>: Invalid type annotation '<instance of tuple>'  [invalid-annotation]
File "tuples_type_form.py", line 45, in <module>: Invalid type annotation '<instance of tuple>'  [invalid-annotation]
"""
conformance_automated = "Fail"
errors_diff = """
Line 12: Unexpected errors ['File "tuples_type_form.py", line 12, in <module>: Type annotation for t1 does not match type of assignment [annotation-type-mismatch]']
Line 14: Unexpected errors ['File "tuples_type_form.py", line 14, in <module>: Type annotation for t2 does not match type of assignment [annotation-type-mismatch]']
Line 15: Unexpected errors ['File "tuples_type_form.py", line 15, in <module>: Type annotation for t2 does not match type of assignment [annotation-type-mismatch]']
Line 25: Unexpected errors ['File "tuples_type_form.py", line 25, in <module>: Type annotation for t10 does not match type of assignment [annotation-type-mismatch]']
Line 36: Unexpected errors ['File "tuples_type_form.py", line 36, in <module>: Type annotation for t20 does not match type of assignment [annotation-type-mismatch]']
Line 40: Unexpected errors ['File "tuples_type_form.py", line 40, in <module>: Invalid type annotation \\'tuple[int, int, Any]\\'  [invalid-annotation]']
Line 41: Unexpected errors ['File "tuples_type_form.py", line 41, in <module>: Invalid type annotation \\'Ellipsis\\'  [invalid-annotation]']
Line 42: Unexpected errors ['File "tuples_type_form.py", line 42, in <module>: Invalid type annotation \\'tuple[Any, int]\\'  [invalid-annotation]', 'File "tuples_type_form.py", line 42, in <module>: Invalid type annotation \\'Ellipsis\\'  [invalid-annotation]']
Line 43: Unexpected errors ['File "tuples_type_form.py", line 43, in <module>: Invalid type annotation \\'tuple[int, Any, int]\\'  [invalid-annotation]', 'File "tuples_type_form.py", line 43, in <module>: Invalid type annotation \\'Ellipsis\\'  [invalid-annotation]']
Line 44: Unexpected errors ['File "tuples_type_form.py", line 44, in <module>: Invalid type annotation \\'<instance of tuple>\\'  [invalid-annotation]']
Line 45: Unexpected errors ['File "tuples_type_form.py", line 45, in <module>: Invalid type annotation \\'<instance of tuple>\\'  [invalid-annotation]']
"""
